documentation/migration-guides/Stack/migration-guide.1.2.12.md

```diff
@@ -0,0 +1,113 @@
+
+- The cmdlets ```*-AzsReourceProviderManifest``` and ```*-AzsUsageConnection``` are deprecated
+For any of these usages, template based resource creation is preferred.  For deletion of these resources, deleting the whole resource group is preferred. If needed, you can also use generic cmdlet *-AzureRmResource
+
+**Parameter Changes**<br>
+The previous release was printing  warnings about the deprecation of the following parameter aliases and they are deprecated now. 
+- Parameter alias ```PlanName``` has been deprecated in favor of ```Name``` in Plan cmdlets
+- Parameter alias ```OfferName``` has been deprecated in favor of ```Name``` in Offer cmdlets
+- Parameter alias ```ResourceGroup``` has been deprecated in favor of ```ResourceGroupName``` in all the cmdlets
+- Parameter ```Managed``` in Get-AzsPlan has been deprecated. It was not used before
+- Parameter ```Managed``` in Get-AzsOffer has been deprecated. Please use Get-AzsManagedOffer instead
+
+**Flattening of Properties** <br>
+The Plan/Offer/Subscription objects wont be having .Properties instead the child properties are flattened out to be at the Top level. If you are having any references to the .Properties, this breaking change could be fixed by accessing the prperty directly at the top level
+
+**Force Parameter** <br>
+
+Remove-* cmdlets will ask for confirmation before doing the remove action. Please use -Force to avoid the confirmation. The following are the affected cmldets
+- Remove-AzsOffer
+- Remove-AzsPlan
+- Remove-AzsUserSubscription
+- Remove-AzsSubscription
+
+<br>
+
+## Breaking Changes in Storage Admin Module
+
+**Globally Removed Parameters**<br>
+The parameters ```SkipCertificateValidation``` and ```DefaultProfile``` have been removed from all cmdlets.
+
+The parameters ```TimeGrain```, ```StartTimeInUtc```, ```EndTimInUtc```, ```MetricNames```, ```DetailedOutput```  have been removed for Get-Azs*Metric Cmdlets, please replace with Where-Object and Select-Object.
+
+The pareamters ```MetricNames```, ```DetailedOutput```  have been removed for Get-Azs*MetricDefinition Cmdlets, please replace with Where-Object and Select-Object
+
+**New Parameters**
+- The parameter ```ResourceGroupName``` is now an optional for all cmdlets.
+- The parameter ```FarmName``` will be required for all Cmdlets that reference nested resources under a farm.  You can get the value using the following
+
+```powershell
+        (Get-AzsStorageFarm -ResourceGroupName $rgn).Name
+```
+Affected cmdlets
+- Get-AzsBlobService
+- Get-AzsBlobServiceMetric
+- Get-AzsBlobServiceMetricDefinition
+- Get-AzsStorageFarmMetric
+- Get-AzsStorageFarmMetricDefinition
+- Get-AzsStorageShare
+- Get-AzsStorageShareMetric
+- Get-AzsStorageShareMetricDefinition
+- Get-AzsTableService
+- Get-AzsTableServiceMetric
+- Get-AzsTableServiceMetricDefinition
+- Start-AzsReclaimStorageCapacity
+
+
+-  ```ResourceId``` parameter can be used for cmdlets that require ```Name``` 
```

Comment:
Can you elaborate on this?  When I first read this I didn't understand right away.  Also, you should mention this has an Alias for 'id'.

Reply:
added a line. agreed this is confusing, will be better if we add an example. I think we will have to do that later. Unlike subscription module, storage module was not used extensively by the customer